components/FlexibleHeader/src/MDCFlexibleHeaderView+ShiftBehavior.h

```diff
@@ -129,6 +129,9 @@ typedef NS_ENUM(NSInteger, MDCFlexibleHeaderContentImportance) {
 /** Asks the receiver to take the header off-screen if it's currently on-screen. */
 - (void)shiftHeaderOffScreenAnimated:(BOOL)animated;
 
+/** Indicates whether the header is presently shifted offscreen. */
+@property(nonatomic, readonly) BOOL shiftedOffscreen;
```

Comment:
Tip: Consider naming the getter, `isShiftedOffscreen`.

Reply:
Done.